WordPress/src/main/java/org/wordpress/android/ui/debug/cookies/DebugCookiesFragment.kt

```diff
@@ -0,0 +1,71 @@
+package org.wordpress.android.ui.debug.cookies
+
+import android.os.Bundle
+import android.view.View
+import android.widget.EditText
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
+import dagger.android.support.DaggerFragment
+import org.wordpress.android.R
+import org.wordpress.android.databinding.DebugCookiesFragmentBinding
+import javax.inject.Inject
+
+class DebugCookiesFragment : DaggerFragment(R.layout.debug_cookies_fragment) {
```

Comment:
DaggerFragment use might make it hard when upgrading to Fragment or another DI,  unless it is really required to be used it here

Reply:
Good point! Updated in f851ab819427ff710f83f325c9f2c3b9639ffe04.